src/main/java/org/jabref/gui/JabRefGUI.java

```diff
@@ -37,7 +37,7 @@ public class JabRefGUI {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(JabRefGUI.class);
 
-    private static JabRefFrame mainFrame;
+    private static  JabRefFrame mainFrame;
```

Comment:
It adds spaces in front?

Reply:
no that was a manual fix, it added a final here that would lead to compile errors